src/app/services/subreddit.service.ts

```diff
@@ -15,4 +20,10 @@ export class SubredditService {
     getAllSubreddits(): Observable<Subreddit[]> {
         return this.subreddits$;
     }
+
+    createSubreddit(subreddit: SubredditCreation): Observable<SubredditCreation> {
+        return this._httpClient.post(`${this._baseUrl}subreddits/create`, subreddit) as Observable<
```

Comment:
I dont see the token in the header?

Reply:
We have discussed this issue an it is handled in BE